File: AndroidCommonUtils/src/main/java/com/fengyongge/androidcommonutils/ToastUtils.java
package com.fengyongge.androidcommonutils;

import android.content.Context;
import android.widget.TextView;
import android.widget.Toast;

import java.util.List;

/**
 * describe
 * toast
 * @author fengyongge(fengyongge98@gmail.com)
 * @version V1.0
 * @date 2019/6/1
 */

public final class ToastUtils {

	public static Toast toast;

	public static void showToast(Context context, String content){
		if(toast==null){
			toast = Toast.makeText(context,content, Toast.LENGTH_SHORT);
		}else{
			toast.setText(content);
		}
		toast.show();
	}


}

File: AndroidCommonUtils/src/main/java/com/fengyongge/androidcommonutils/ktutils/ToastUtils.kt
package com.fengyongge.androidcommonutils.ktutils

import android.content.Context
import android.widget.Toast
import com.fengyongge.androidcommonutils.AndroidCommonUtils

/**
 * describe
 *
 * @author fengyongge(fengyongge98@gmail.com)
 * GitHub: https://github.com/fengyongge/android-commonutils
 * @date 2020/09/08
 */
class ToastUtils {

    companion object{
        fun showToast(context: Context?,messsage: String?){
            Toast.makeText(context,messsage,Toast.LENGTH_SHORT).show()
        }

        fun showToast(messsage: String?){
            Toast.makeText(AndroidCommonUtils.getAppContext(),messsage,Toast.LENGTH_SHORT).show()
        }
    }

}